services/sharedchannel/attachment.go

```diff
@@ -156,7 +157,7 @@ func (scs *Service) onReceiveUploadCreate(msg model.RemoteClusterMsg, rc *model.
 	us.RemoteId = rc.RemoteId // don't let remotes try to impersonate each other
 
 	// create upload session.
-	usSaved, appErr := scs.app.CreateUploadSession(&us)
+	usSaved, appErr := scs.app.CreateUploadSession(request.EmptyContext(), &us)
```

Comment:
Same as earlier, we don't need to pass `EmptyContext()` in code like these.

Reply:
As earlier, this calls a method requiring the context: `GetChannel()`. `GetChannel` is clearly part of request flows, just not in this particular instance.